groups/bdl/bdlb/bdlb_guidutil.cpp

```diff
@@ -124,18 +124,26 @@ Guid GuidUtil::generate()
     return result;
 }
 
-void GuidUtil::generate(unsigned char *result, bsl::size_t numGuids)
+namespace {
+inline
+void makeRfc4122Compliant(unsigned char *bytes, unsigned char *end)
```

Comment:
Move this function into the anonymous namespace at the beginning of the file:  we always put such implementation helper functions before implementation of any functions from the header.